distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Vector_Builder.enso

```diff
@@ -45,6 +45,12 @@ type Vector_Builder
     is_empty : Boolean
     is_empty self = self.length == 0
 
+    ## PRIVATE
+
+       Checks if this builder is not empty.
+    not_empty : Boolean
+    not_empty self = self.is_empty.not
```

Comment:
do we needs this? This feels slightly like polluting API

Reply:
We have this on `Vector` and `Text` so more like making the API consistent. But I agree it is a bit redundant.

Reply:
Just to show it :sweat_smile: 
![image](https://user-images.githubusercontent.com/1436948/210415819-9e42c255-3293-45a3-bb52-f792681cd8af.png)

Reply:
I guess we should be consistent though - either have a `not_empty` for each `is_empty`, or none at all.